### PR TITLE
multiple fixes

### PR DESCRIPTION
**Pull request by gitlawr**
This PR address multiple issues:

1. Support updaing draft(https://github.com/seal-io/walrus/issues/1462, https://github.com/seal-io/walrus/issues/1461)
2. Add inactive resource count to env list API(https://github.com/seal-io/walrus/issues/1469). This is a temporary fix before https://github.com/seal-io/walrus/issues/1500 is addressed.
3. Fix scheduling resources to be stopped.(https://github.com/seal-io/walrus/issues/1491)

See commit message for details.

### CODE REVIEW

**Review by shanewxy — APPROVED** [review]
LGTM

File: pkg/scheduler/resource/relationship_check_task.go [hunk 1]
```diff
@@ -162,6 +167,39 @@ func (in *RelationshipCheckTask) destroyResources(ctx context.Context) error {
...
 }
 
+// stopResources stops all resources that are in the progressing and stopping state.
+func (in *RelationshipCheckTask) stopResources(ctx context.Context) error {
+	resources, err := in.modelClient.Resources().Query().
+		Where(
+			func(s *sql.Selector) {
+				s.Where(sqljson.ValueEQ(
+					resource.FieldStatus,
+					summaryStatusProgressing,
```
Comment (alexcodelf) [inline pkg/scheduler/resource/relationship_check_task.go:177]: Is the summary status for stopping resources is `Progressing`?

Comment (gitlawr) [reply pkg/scheduler/resource/relationship_check_task.go:177]: When it's waiting for dependencies, it's in `progressing` with the dependency message. When dependencies are resolved, it turns to `stopping` the job runs under the hood.

**Review by alexcodelf — APPROVED** [review]
LGTM

**Pull request merged**